highlighters/main/test-data/alias-precommand-option-argument1.zsh

```diff
@@ -35,7 +35,6 @@ BUFFER='sdu phy1729 echo foo'
 
 expected_region_highlight=(
   '1 3 alias' # sdu
-  '1 3 precommand' # sdu (sudo)
   '5 11 default' # phy1729
   '13 16 commmand "issue #540"' # echo (not builtin)
```

Comment:
Note that if `echo` ceased being highlighted as `builtin`, the test wouldn't fail. Can we use, say, `ls` here to test this situation, and add a separate XFail test for the second word of `sudo echo` (without aliases)?

Reply:
Added alias-precommand-option-argument3.